Tests/js/apps/assets/ez-platformuiapp-tests.js

```diff
@@ -314,7 +314,30 @@ YUI.add('ez-platformuiapp-tests', function (Y) {
                 dispatchedEvent,
                 "The `ez:yui-app:ready` event should have been dispatched"
             );
-        }
+        },
+
+        // Regression test for https://jira.ez.no/browse/EZP-27421
+        "Should dispatch the `ez:yui-app:ready` event after plugins are initialized": function () {
+            var app,
+                pluginNS = 'myPlugin',
+                dispatchedEvent = false,
+                Plugin = Y.Base.create('myPlugin', Y.Plugin.Base, [], {
+                    initializer: function () {
+                        this.on('initializedChange', function () {
```

Comment:
also, I realize that in the previous test and in this one, the event listener for `ez:yui-app:ready` is never removed which at some point will cause weird bugs. Can you add a call to `document.removeEventListener` in both test to clean up that ?